src/app/analytics/analytics.factory.js

```diff
@@ -537,9 +537,15 @@ const analyticsFactory = /* @ngInject */ function ($window, $timeout, sessionSer
               category: cartItem.designationType.toLowerCase(),
               variant: cartItem.frequency.toLowerCase(),
               quantity: '1',
-              dimension3: cartItem.frequency.toLowerCase() === 'single' ? 'one-time' : 'recurring',
+              dimension1: localStorage.getItem('gaDonorType'),
+              dimension2: localStorage.getItem('gaDonorAcct') || undefined,
+              dimension3: cartItem.designationType.toLowerCase(),
               dimension4: cartItem.frequency.toLowerCase(),
-              dimension8: cartItem.designationType.toLowerCase()
+              dimension5: cartItem.frequency.toLowerCase() === 'single' ? 'one-time' : 'recurring',
```

Comment:
I'm not a fan of the wording between 4 and 5. I would have swapped these if I were interpreting them. Are we sure this is the correct mapping?

Reply:
Yeah same. I believe four is just supposed to be what the frequnce value is while five is supposed to confirm if it's a reoccurring type or not. I'll double check but I believe this is what they wanted.